### ISSUE TITLE
Accessibility options

### ISSUE THREAD

**Issue by RenderWarrior** [WPDreams/ajax-search-pro-development#230]
### Is your feature request related to a problem? Please describe.
I'm working os some customers sites for improve the Accessibility, as it's required by a EU law.
I noticed the search field has no label (and that's marked as error by our accessibility checker)

### Describe the solution you'd like
Add an option for includina a label tag

**Comment by ernestmarcinko** [comment]
Hi,

The search input has an [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label) attribute, and it's value can be adjusted via the back-end options. (form, input, autocomplete, settings form and magnifier button aria-labels).
Using _aria-label_ is [WCAG compliant](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA14) in cases where an actual label element is not practical, such as this case.

I will however consider adding an [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby) option to the back-end for each form/input in case an external element is desired to be used as the label.
